### DISCUSSION TITLE
Welcome to RODA Discussions!

### DISCUSSION THREAD

**Discussion by luis100** [keeps/roda#2639] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Josefine82** [comment]
Hi! 
I am taking a course in digital preservation systems. One of the study assignment includes testing different tools for preservation.
I hope to learn more about RODA through participate in the discussions. Is there a tutorial video for RODA somewhere? 
/Josefine

**Reply by luis100** [reply]
Check if this suits you: https://training.eark.online/course/view.php?id=6#section-1

**Reply by Josefine82** [reply]
Perfect! Thanks a lot.
/Josefine
